src/main/java/uk/gov/ch/developer/docs/DocsWebApplication.java

```diff
@@ -28,4 +32,15 @@ public static void main(String[] args) {
     public void addInterceptors(InterceptorRegistry registry) {
         registry.addInterceptor(loggingInterceptor);
     }
+    
+    @Bean
+    public WebServerFactoryCustomizer<TomcatServletWebServerFactory> webServerFactoryCustomizer() {
```

Comment:
As raised in offline discussion, this may not be needed.
If it is, could we have it documented why we wanted to override our servers default cookie processor?